src/components/Home.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable function-paren-newline */
+/* eslint-disable implicit-arrow-linebreak */
+/* eslint-disable react/jsx-one-expression-per-line */
+import React, { useEffect, useState } from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+import { getMovies } from './redux/MovieSlice';
+import Movie from './Movie';
+import Header from './Header';
+import NotFound from './NotFound';
+
+function Home() {
+  const moviesList = useSelector((state) => state.movies.movies);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getMovies());
+  }, [dispatch]);
+
+  const a = moviesList[0];
+  const b = a?.results;
+
+  const [search, setSearch] = useState('');
+
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const filtered = b?.filter((m) =>
+    m.original_title.toLowerCase().includes(search.toLowerCase()),
+  );
+
+  if (moviesList.length === 0) {
+    return (
+      <div className="loading-container">
+        <Header />
+        <div className="circle" />
+        <h1 className="loading">Loading...</h1>
+      </div>
+    );
+  }
+
+  return (
+    <div className="home-container">
+      <Header search={search} handleSearch={handleSearch} />
+      <div className="main-moviesList-container">
+        <div className="moviesList-container">
+          {filtered?.map((m) => (
+            <Movie movie={m} key={m.id} id={m.id} />
+          ))}
+          ,{filtered?.length === 0 && <NotFound />}
+        </div>
+      </div>
+    </div>
+  );
```

Comment:
- [ ] I would like to reiterate what the other reviewers have been highlighting. Your application doesn't have the top section. Remember that one of the requirements for this project is to make your application design identical to the wireframe provided. Kindly implement this change. 

![Screenshot-Today](https://user-images.githubusercontent.com/77106962/211147335-d373f250-8521-4180-bf2a-a96277da7ed4.png)

#### Hint: You can do either of the two:

- Get a general movie logo from Google, use the image for the top section and name it Movies App or a name you prefer (preferable) or

- Use one of the item's images for the top section and name it whatever you wish. 👍 💻

Reply:
Fixed.